tests/integration_tests/cache_tests.py

```diff
@@ -46,7 +46,7 @@ def test_no_data_cache(self):
         app.config["DATA_CACHE_CONFIG"] = {"CACHE_TYPE": "NullCache"}
         cache_manager.init_app(app)
 
-        slc = self.get_slice("Girls", db.session)
+        slc = self.get_slice("Top 10 Girl Name Share", db.session)
```

Comment:
Switching out for a legacy chart which invokes the `/superset/explore_json` RESTful API endpoint.